src/gdbstub_impl/ext/exec_file.rs

```diff
@@ -0,0 +1,28 @@
+use super::prelude::*;
+use crate::protocol::commands::ext::ExecFile;
```

Comment:
nit: add space between different import groups (i.e: `super` and `crate`)

Reply:
Oh no, you may need to take a look at other files in `src/gdbstub_impl/ext`😅

Reply:
ahhh, yep, fair enough. not sure why past me thought that was a good idea, but alright...